include/RAJA/policy/hip/raja_hiperrchk.hpp

```diff
@@ -25,18 +25,97 @@
 #if defined(RAJA_ENABLE_HIP)
 
 #include <iostream>
+#include <utility>
+#include <algorithm>
+#include <tuple>
+#include <array>
+#include <string_view>
 #include <string>
+#include <sstream>
+#include <stdexcept>
 
 #include <hip/hip_runtime.h>
 
+#include "camp/defines.hpp"
+#include "camp/helpers.hpp"
+
 #include "RAJA/util/macros.hpp"
 
+#if defined(__HIPCC__)
+#define ROCPRIM_HIP_API 1
```

Comment:
I don't see this used anywhere. Remove?

Reply:
Let me see if I can remember what this was for, hopefully it can just go away.

Reply:
I think his was something we had to define to tell rocprim what we wanted to exist in their headers.

Reply:
So it's needed, and in more than one file? That seems strange.

Reply:
I define it before including any rocprim files just in case, we could guard defining it more than once, but I haven't seen any warnings.

Reply:
Maybe add a comment if needed?